src/lib.rs

```diff
@@ -477,3 +492,87 @@ impl fmt::Display for KsuidMs {
         write!(f, "{}", self.to_base62())
     }
 }
+
+#[cfg(feature = "serde")]
+impl Serialize for Ksuid {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        serializer.serialize_str(&self.to_base62())
+    }
+}
+
+#[cfg(feature = "serde")]
+struct KsuidVisitor;
+
+#[cfg(feature = "serde")]
+impl<'de> Visitor<'de> for KsuidVisitor {
+    type Value = Ksuid;
+
+    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+        formatter.write_str("a base62 &str")
```

Comment:
Maybe this should be more specific?

Reply:
let me digest what this method actually does and come back with improvement ;)

Reply:
I also have no idea, I thought it was just a user-visible description, but maybe not?

Reply:
so the current implementation in case if some error occurs will result in: `This Visitor expects to receive a base62 &str.`

ref:
> This is used in error messages. The message should complete the sentence “This Visitor expects to receive …”, for example the message could be “an integer between 0 and 64”. The message should not be capitalized and should not end with a period.

https://docs.rs/serde/latest/serde/de/trait.Visitor.html#tymethod.expecting

I'd leave it as is but if you would like different error message pls propose some alternative

Reply:
"a valid ksuid in bas62"?